esmvaltool/config-developer.yml

```diff
@@ -262,26 +262,26 @@ ana4mips:
   input_dir:
     default: 'Tier[tier]/[dataset]'
   input_file: '[short_name]_[mip]_[type]_[dataset]_*.nc'
-  output_file: '[project]_[mip]_[type]_[dataset]_[field]_[short_name]_[start_year]-[end_year]'
+  output_file: '[project]_[mip]_[type]_[dataset]_[short_name]_[start_year]-[end_year]'
   cmor_type: 'CMIP5'
 
 CCMVal1:
   input_dir:
     default: '[exp]/[dataset]'
-  input_file: '[project]_[exp]_[name]_[ensemble]_[field]_[short_name].nc'
-  output_file: '[project]_[exp]_[name]_[ensemble]_[field]_[short_name]_[start_year]-[end_year]'
+  input_file: '[project]_[exp]_[name]_[ensemble]_[mip]_[short_name].nc'
+  output_file: '[project]name]_[ensemble]_[short_name]_[start_year]-[end_year]'
```

Comment:
`CCMVal1` actually uses the field to name the input files, so it should not be changed.
But we can probably remove this project (and `CCMVal2` as well), since it is quite old and no longer used.

Reply:
Ok, I'll remove it